spec/locales-currencies-tz.html

```diff
@@ -107,6 +107,9 @@ <h1>DefaultLocale ( ): a Unicode canonicalized locale identifier</h1>
         <dt>description</dt>
         <dd>The returned String value represents the structurally valid (<emu-xref href="#sec-isstructurallyvalidlanguagetag"></emu-xref>) and canonicalized (<emu-xref href="#sec-canonicalizeunicodelocaleid"></emu-xref>) language tag for the host environment's current locale. It must not contain a Unicode locale extension sequence.</dd>
       </dl>
+      <emu-note>
+        In browser environments, this value is derived from <code>navigator.languages</code>, which could potentially be used as a fingerprinting vector. In non-browser environments, it is generally determined by the host environment's locale settings.
```

Comment:
"derived from" doesn't seem right to me, rather `navigator.language` and DefaultLocale should agree with each other and **both** derive from the same root source.
```suggestion
        The returned value is is a potential fingerprinting vector. In browser environments, it should match <a href="https://html.spec.whatwg.org/#language-preferences"><code>navigator.language</code></a> to avoid providing any additional distinguishing information.
```